.sdk_metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "sdks": {
+    "swift-client-sdk": {
+      "name": "iOS SDK",
+      "type": "client-side",
+      "languages": [
+        "Swift"
+      ]
+    }
+  }
+}
```

Comment:
So, I've given this the primary key `swift-client-sdk` because I know we have been thinking about referring to it like that for a while. This isn't actually user facing, just the key used in the metadata database.

We can eventually update the `name` if we want, too.

Finally, I've put `Swift` as the language - should it have `Objective-C` too?

Reply:
I think the objective c layer is generated.  https://github.com/launchdarkly/ios-client-sdk/blob/v9/LaunchDarkly/LaunchDarkly/ObjectiveC/ObjcLDClient.swift